README.md

```diff
@@ -1,43 +1,135 @@
+| **CesiumGltf**                 | Lightweight glTF processing and optimization functions.                                                         |
+| **CesiumGltfContent**          | Classes that support manipulating the content of a glTF.                                                        |
+| **CesiumGltfReader**           | glTF deserialization / decoding, including glTF extension support (`KHR_draco_mesh_compression` etc).           |
+| **CesiumGltfWriter**           | glTF serialization / encoding, including glTF extension support.                                                |
+| **CesiumIonClient**            | Functions to access [Cesium ion](https://cesium.com/cesium-ion/) accounts and 3D tilesets using ion's REST API. |
+| **CesiumJsonReader**           | Reads JSON from a buffer into statically-typed classes.                                                         |
+| **CesiumJsonWriter**           | Writes JSON from statically-typed classes into a buffer.                                                        |
+| **CesiumQuantizedMeshTerrain** | Classes for accessing terrain in the [quantized-mesh-1.0](https://github.com/CesiumGS/quantized-mesh) format.   |
+| **CesiumRasterOverlays**       | Classes for raster overlays, which allow draping massive 2D textures over a model.                              |
+| **CesiumUtility**              | Utility functions for JSON parsing, URI processing, credits, etc.                                               |
 
 
 ### 📗License
 
 [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0.html). Cesium Native is free for both commercial and non-commercial use.
 
+### 🔧Projects Using Cesium Native
+
+The following official Cesium integrations are built on top of Cesium Native:
+
+- [Cesium for Unreal](https://github.com/CesiumGS/cesium-unreal)
+- [Cesium for Unity](https://github.com/CesiumGS/cesium-unity)
+- [Cesium for Omniverse](https://github.com/CesiumGS/cesium-omniverse)
+- [Cesium for O3DE](https://github.com/CesiumGS/cesium-o3de)
+
+In addition, Cesium Native has been used by the community to build projects including:
+- [vsgCs](https://github.com/timoore/vsgCs) integrates Cesium Native with VulkanSceneGraph.
+- [osgEarth](https://github.com/gwaldron/osgearth) uses Cesium Native to load 3D Tiles in OpenSceneGraph.
+
+If you have a project that integrates with Cesium Native that isn't on this list yet, please let us know!
+
+### 📦Dependencies
+
+Cesium Native relies on various third-party packages to accomplish tasks like image decoding, URI parsing, and math operations. These dependencies are automatically obtained and built using [vcpkg](https://vcpkg.io/en/) when building through CMake. Below is a graph of all the Cesium Native libraries and their dependencies on third party packages and each other.
+
+![](doc/img/dependency-graph.svg)
+
 ## 💻Developers
```

Comment:
I kind of like what we did in Cesium for Unreal's README where we point developers to a more dedicating landing page for development: https://github.com/CesiumGS/cesium-unreal?tab=readme-ov-file#computer-developing-with-unreal-engine

That might be personal preference though -- I like compact READMEs. 😛 Does anyone else have opinions?